kedro/extras/datasets/tensorflow/tensorflow_model_dataset.py

```diff
@@ -118,15 +118,17 @@ def _load(self) -> tf.keras.Model:
                 self._fs.get(load_path, path, recursive=True)
 
             # Pass the local temporary directory/file path to keras.load_model
-            return tf.keras.models.load_model(path, **self._load_args)
+            device = self._load_args.pop("tf_device", "gpu")
```

Comment:
Nit: I'm inclined to call the argument `d
```suggestion
            device_name = self._load_args.pop("device_name", None)
```

Reply:
I have updated the variable name